projects/plugins/boost/app/assets/src/js/api/speed-scores.ts

```diff
@@ -41,15 +50,15 @@ export async function clearCache(): Promise< void > {
  *
  * @return {SpeedScores} Speed scores returned by the server.
  */
-export async function requestSpeedScores(): Promise< SpeedScores > {
+export async function requestSpeedScores(): Promise< SpeedScoresSet > {
 	// Request metrics
 	const response = parseResponse(
 		await api.post( '/speed-scores', { url: Jetpack_Boost.site.url } )
 	);
 
 	// If the response contains ready-to-use metrics, we're done here.
 	if ( response.scores ) {
-		return response.scores;
+		return getResponseSpeedScoresSet( response );
```

Comment:
This demonstrates why the getter isn't necessary; it uses `response.scores` immediately before a call to `getResponseSpeedScoresSet`, requiring future developers to look into what `getResponseSpeedScoresSet` does which `response.scores` doesn't.